README.md

```diff
@@ -1,218 +1,191 @@
 # AWS CDK RFCs
 
-This repo is a place to propose and track upcoming changes to [AWS CDK], [jsii], and
```

Comment:
what happened here? revert please

Reply:
A reflow happened

Reply:
Please revert. The table is auto-generated